source/code-snippets/authentication/aws.js

```diff
@@ -5,12 +5,17 @@ const accessKeyId = encodeURIComponent("<AWS_ACCESS_KEY_ID>");
 const secretAccessKey = encodeURIComponent("<AWS_SECRET_ACCESS_KEY>");
 const clusterUrl = "<MongoDB cluster url>";
 
+const sessionToken = encodeURIComponent("<AWS_SESSION_TOKEN>"); // Optional
+
 const authMechanism = "MONGODB-AWS";
 
 // Replace the following with your MongoDB deployment's connection string.
 const uri =
   `mongodb+srv://${accessKeyId}:${secretAccessKey}@${clusterUrl}/?authMechanism=${authMechanism}`;
 
+// If you created a session token, append to the end of your URI string
+const tokenSetting = '&authMechanismProperties=${sessionToken}';
```

Comment:
A couple items here:

1) I think for string interpolation (i.e. using the ${} syntax to parse a value into a string) you need backticks similar to what's on line 14. Also, we should probably show how to append it rather than storing it in a variable that doesn't get used. 

2) The comment should answer the question of when they need it more precisely. While you can create a session token on AWS, that is independent of whether you need it for authentication. Instead, how about something like "Uncomment the following line if your AWS authentication setup requires a session token." and comment out this line.

Putting these together, your code might look something like this:
```
// Uncomment the following line if your AWS authentication setup requires a session token 
// uri.concat(`&authMechanismProperties=${sessionToken}`);
```

Reply:
accepted. changed to reflect that concat does not change strings in place. I also had to use "let" to initialize because const types are not editable.